pkgs/applications/misc/mupdf/default.nix

```diff
@@ -37,41 +36,28 @@
 , xcbuild
 , gitUpdater
 
-# for passthru.tests
+  # for passthru.tests
 , cups-filters
 , zathura
 , mupdf
 }:
 
 assert enablePython -> enableCxx;
 
-let
-
-  freeglut-mupdf = freeglut.overrideAttrs (old: rec {
```

Comment:
I'd be happy to remove this in my PR as well. What made you remove it?

Reply:
The repository was not updated in 3 years, so I want to know if this is really necessary or not.

Reply:
Looking at the Arch package, they seem to be using `freeglut` from upstream too.